src/messages/responses/topic-item.ts

```diff
@@ -0,0 +1,43 @@
+import {truncateString} from '../../internal/utils/display';
+
+/**
+ * Represents the data received from a topic subscription.
+ *
+ * @remarks A subscription is created by calling {@link TopicClient.subscribe}.
+ * The value is guaranteed to be either a {@link string} or a {@link Uint8Array}.
+ * Call the appropriate accessor if you know the type of the value.
+ */
+export class TopicItem {
+  private readonly _value: string | Uint8Array;
+  constructor(_value: string | Uint8Array) {
+    this._value = _value;
+  }
+  /**
+   * Returns the data read from the stream.
+   * @returns string | Uint8Array
+   */
+  public value(): string | Uint8Array {
```

Comment:
I'm going to submit an issue on this repo to start adding a `json()` option for values that automatically parses the value and returns a json object. It's a nice to have feature I've seen in other places like Postman that stands out as great DX

Reply:
Sounds interesting. Can you share a link?